src/helpers/validator.js

```diff
@@ -0,0 +1,16 @@
+import Joi from 'joi';
+
+const Schema = {
+  login: Joi.object().keys({
+    email: Joi.string()
+      .email()
+      .required()
+      .error(new Error('Please fill all fields')),
+    password: Joi.string()
+      .required()
+      .trim()
+      .error(new Error('Please fill all fields')),
```

Comment:
add more validation, Capital, number, and length. As well as handle the error message correctly

Reply:
These were handled on user signup, I thought I only had to validate if the password a user puts in matches the one he used on signup